Sources/StreamChat/Config/ChatClientConfig.swift

```diff
@@ -83,20 +83,19 @@ extension ChatClientConfig {
     }
 }
 
-// extension ChatClientConfig {
-//    /// `ChatChannel` specific settings.
-//    public struct Channel {
-//        // example ...
-//        public var isReplyInChannelAllowed = true
-//
-//        /// When `KeystrokeEvent` is sent, the time interval before the `TypingStop` event is automatically sent.
-//        public var keystrokeEventTimeout: TimeInterval = 5
-//    }
-//
+extension ChatClientConfig {
+    /// `ChatChannel` specific settings.
+    public struct Channel {
+        /// Limit how many watchers are returned in `_ChatChannel.lastActiveWatchers`
+        public var lastActiveWatchersLimit = 25
+        /// Limit how many members are returned in `_ChatChannel.lastActiveMembers`
+        public var lastActiveMembersLimit = 25
+    }
+
```

Comment:
I think these 2 should live in `ChannelController` as params, maybe `ChannelQuery` even (they already are present in `ChannelQuery`